### ISSUE TITLE
Simpleblogger cant start on EL7 - Namespace WebKit2 not available

### ISSUE THREAD

**Issue by ivoarch** [e-shulitsky/simpleblogger#3]
Terminal output:

```
~]$ simpleblogger 
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/simpleblogger/simpleblogger.py", line 14, in <module>
    import utils
  File "/usr/lib/python2.7/site-packages/simpleblogger/utils.py", line 23, in <module>
    from simpleblogger import CONFIG_DIRECTORY
  File "/usr/lib/python2.7/site-packages/simpleblogger/simpleblogger.py", line 18, in <module>
    from dialogs.preview import PreviewWindow
  File "/usr/lib/python2.7/site-packages/simpleblogger/dialogs/preview.py", line 4, in <module>
    gi.require_version('WebKit2', '4.0')
  File "/usr/lib64/python2.7/site-packages/gi/__init__.py", line 81, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace WebKit2 not available
```

**Comment by jsnjack** [comment]
Thank you reporting the bug. I think one of the dependencies is missing. Could you please check if you have webkitgtk4-devel.x86_64 or/and webkitgtk4.x86_64 installed?

**Comment by ivoarch** [comment]
Hi,
For the CentOS 7 there are only `webkitgtk{,-devel}` , `webkitgtk3{,-devel}` packages from EPEL repo . I have installed them and the output now is  `ValueError: Namespace WebKit2 not available for version 4.0` .

**Comment by jsnjack** [comment]
You can probably try to install webkitgtk4 package from the source (not sure, how easy it is) or you can edit the source code and replace `gi.require_version('WebKit2', '4.0')` with `gi.require_version('WebKit2', '3.0')` and see what happens
